### DISCUSSION TITLE
Again no taskbar mods after today's Windows update?

### DISCUSSION THREAD

**Discussion by ygoe** [ramensoftware/windhawk-mods#643] [Q&A] [answered]
Hello,

I always wait till the end of the month to do my Windows updates so that the symbols are available and everybody has updated things to work with whatever Microsoft has changed again. The recent update on my laptop went fine and after the reboot, symbols for explorer.exe were downloaded.

Now on my PC, no taskbar mods are running. No symbols download, no errors, no updates.

Does anybody know what the issue is? I only agreed to use Windows 11 because this tool and its useful mods exist. Without them, I'd rather go back to Windows 10 and 7+TT.

**Comment by m417z** [comment, marked as answer]
The short version is that Microsoft decided to remove all debugging symbols for explorer.exe, taskbar.dll, taskbar.view.dll, and maybe others. I'll publish a more detailed announcement later. For now, you can use this workaround: https://github.com/ramensoftware/windhawk-mods/issues/641#issuecomment-2092499571

**Comment by ygoe** [comment]
Oh dear. Have to watch this closely before doing any Windows updates again!

**Reply by m417z** [reply]
I posted more details here:
https://ramensoftware.com/windhawk-and-symbol-download-errors
